bower.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/PolymerElements/paper-menu",
   "ignore": [],
   "dependencies": {
-    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^0.9.0",
+    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^0.9.2",
```

Comment:
You'll probably want to update this bower.json's version to 0.9.3, if you want to tag a release after merging this :)